pkg/ocm/ocmWrapper.go

```diff
@@ -35,19 +37,51 @@ type OCMInterface interface {
 	GetClusterActiveAccessRequest(ocmConnection *ocmsdk.Connection, clusterID string) (*acctrspv1.AccessRequest, error)
 	CreateClusterAccessRequest(ocmConnection *ocmsdk.Connection, clusterID, reason, jiraIssueID, approvalDuration string) (*acctrspv1.AccessRequest, error)
 	CreateAccessRequestDecision(ocmConnection *ocmsdk.Connection, accessRequest *acctrspv1.AccessRequest, decision acctrspv1.DecisionDecision, justification string) (*acctrspv1.Decision, error)
+	SetupOCMConnection() (*ocmsdk.Connection, error)
 }
 
 const (
-	ClustersPageSize = 50
+	ClustersPageSize      = 50
+	ocmNotLoggedInMessage = "Not logged in"
 )
 
-type DefaultOCMInterfaceImpl struct{}
+type DefaultOCMInterfaceImpl struct {
+	//	connection *ocmsdk.Connection
+}
 
 var DefaultOCMInterface OCMInterface = &DefaultOCMInterfaceImpl{}
 
+// SetupOCMConnection setups the ocm connection for all the other ocm requests
+func (o *DefaultOCMInterfaceImpl) SetupOCMConnection() (*ocmsdk.Connection, error) {
+
+	envURL := os.Getenv("OCM_URL")
+	if envURL != "" {
+		// Fetch the real ocm url from the alias and set it back to the ENV
+		ocmURL, err := ocmurls.ResolveGatewayURL(envURL, nil)
+		if err != nil {
+			return nil, err
+		}
+		os.Setenv("OCM_URL", ocmURL)
+		logger.Debugf("reset the OCM_URL to %s", ocmURL)
+	}
+
+	// Setup connection at the first try
+	connection, err := ocmocm.NewConnection().Build()
+	if err != nil {
+		if strings.Contains(err.Error(), ocmNotLoggedInMessage) {
+			return nil, fmt.Errorf("please ensure you are logged into OCM by using the command " +
+				"\"ocm login --use-auth-code --url $ENV\"")
```

Comment:
Is it always true? 
According to ocm login docs 
```
ocm login --help
.....
      --use-auth-code          Login using OAuth Authorization Code. This should be used for most cases where a browser is available. See --use-device-code for remote hosts and containers.
      --use-device-code        Login using OAuth Device Code. This should only be used for remote hosts and containers where browsers are not available. See --use-auth-code for all other sce
```

Reply:
I am not 100% percent sure what would be the use case for --use-device-code.
But I think we can just simply mention to use `ocm login`